services/datamanager/builtin/builtin_test.go

```diff
@@ -350,27 +349,26 @@ func TestModelDeploy(t *testing.T) {
 
 	zipper, err := os.Create(deployedZipFileName)
 	test.That(t, err, test.ShouldBeNil)
+	defer zipper.Close()
 
 	// Add content to zip file.
 	b0 := []byte("text representing model.txt internals.")
-	r0 := bytes.NewReader(b0)
 	b1 := []byte("text representing README.md internals.")
-	r1 := bytes.NewReader(b1)
 	zipWriter := zip.NewWriter(zipper)
```

Comment:
Supernit: could you move b0/b1 instantiation after the file names, then create the zipper, then create the zipwriter + write? Follows a bit of a more natural ordering flow.

Reply:
gotcha.